src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -334,9 +334,11 @@ public abstract class AbstractJavadocMojo extends AbstractMojo {
 
     /**
      * Specify if the Javadoc should operate in offline mode.
+     *
+     * @since 3.5.1
      */
-    @Parameter(defaultValue = "${settings.offline}", required = true, readonly = true)
-    private boolean isOffline;
+    @Parameter(alias = "isOffline", property = "maven.javadoc.offline", defaultValue = "false")
+    private boolean offline;
```

Comment:
as old version was readonly, internal so was not documented - we need not try to preserve old filed name

Reply:
ok.